HomePage.html

```diff
@@ -61,13 +61,226 @@ <h1>Hey there. I'm Mariane <br> <span>I'm a software developer </span> </h1>
+					
+						</div>
+			
+				</section>
+				
+				<section class="project-card">
+					<div class="project-card-image">
+								<img src="Assets/Images/art-printing.png" alt="Gym Website"/>
+						</div>
+					
+						<div class="card-details">
+					
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Typescript</li>
+										<li>Html</li>
+										<li>Java</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
+					
+						</div>
+			
+				</section>
+
+				<section class="project-card">
+						<div class="project-card-image">
+								<img src="Assets/Images/organizer.png" alt="Gym Website"/>
+						</div>
+					
+						<div class="card-details">
+					
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Html</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
+					
+						</div>
+			
+				</section>
+						<section class="project-card">
+					<div class="project-card-image">
+								<img src="Assets/Images/gym-fit.png" alt="Gym Website"/>
+						</div>
+					
+						<div class="card-details">
+					
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Python</li>
+			
+									
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
+					
+						</div>
+			
+				</section>
+						<section class="project-card">
+					<div class="project-card-image">
+								<img src="Assets/Images/art-printing.png" alt="Gym Website"/>
+						</div>
+					
+						<div class="card-details">
+					
+							<h3 class="card-heading">Multi-Post Stories <br> Gain+Glory</h3>
+							<div class="card-languages">
+								<ul class="language-list">
+										<li>Ruby on Rails</li>
+										<li>Css</li>
+										<li>Javascript</li>
+										<li>Html</li>
+								</ul>
+							</div>
+							<button class="project-card-button"> See Project</button>
+							
+						</div>
+						
+					</section>
+					
+				</div>
+				
+			</div>
+			
+			<div class="about-me-page" id="about-me">
+				
+				<section class="about-me-section">
+					<h2>About Me</h2>
+					<p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+						don’t hesitate to contact me.</p>
+					<button class="resume-button">Get My Resume</button>
```

Comment:
[REQUIRED] Kindly ensure that your button states match the template.

|Template|
|-----------|
|![image](https://user-images.githubusercontent.com/62938735/192018711-8fce6d47-a744-49e0-8755-b2634fb467b5.png)|